src/AbstractPathMappingRepository.php

```diff
@@ -278,4 +301,49 @@ protected function createVirtualResource($path = null)
 
         return $resource;
     }
+
+    /**
+     * Transform a relative path into an absolute path.
+     *
+     * @param string $absolutePath
+     *
+     * @return string
+     */
+    protected function makePathRelative($absolutePath)
+    {
+        return Path::makeRelative($absolutePath, $this->baseDirectory);
+    }
+
+    /**
+     * Transform a relative path into an absolute path.
+     *
+     * @param string $relativePath
+     *
+     * @return string
+     */
+    protected function makePathAbsolute($relativePath)
+    {
+        if (0 === strpos($relativePath, 'l:')) {
```

Comment:
Ah I see it can. I would rename this method (and the plural version before) to `resolveRelativePath()`.